Example/Podfile

```diff
@@ -14,7 +14,7 @@ target 'Emission' do
     pod 'Artsy+UIFonts', :git => "https://github.com/artsy/Artsy-OSSUIFonts.git"
   end
 
-  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket']
+  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket', 'RCTImage']
```

Comment:
When I tried wrapping it, it was just a black square. I took a look at our `opaque_image_view` and it seems to assume the `imageURL` should always be changed into a Gemini-friendly URL. I'm a little hesitant to make a lot of changes there because it's used so often elsewhere in emission, but I can if necessary.